Renewal/System/itemInfo_EN.lua

```diff
@@ -123678,7 +123678,7 @@ tbl = {
 		identifiedDisplayName = "Event Ticket",
 		identifiedResourceName = "����",
 		identifiedDescriptionName = {
-			"����� ��� �߰��� ��� ����ϱ� ��� ��??� Ƽ��. �縷���� ���ν��� �𺧷��� ��� NPC���� ����� �ָ� ��� ���� ��� �� ����.",
+			"����� ��� �߰��� ��� ����ϱ� ��� �̺�Ʈ Ƽ��. �縷���� ���ν��� �𺧷��� ��� NPC���� ����� �ָ� ��� ���� ��� �� ����.",
```

Comment:
Didn't even see that this is still untranslated:
Google Translation:
`Event tickets to commemorate the addition of a new town. If you bring it to an NPC named Developer in the desert city of Veins, something good will happen.`